engine/src/main/java/org/terasology/rendering/logic/SkeletonRenderer.java

```diff
@@ -18,6 +18,10 @@
 
 import com.google.common.collect.Lists;
 import com.google.common.collect.Maps;
+import jdk.nashorn.internal.scripts.JO;
```

Comment:
Accidently import? I suppose using some JDK internal scripts would be a bad idea...
```suggestion
```